api/api.proto

```diff
@@ -80,16 +80,34 @@ message DeletePasswordResp {
   bool not_found = 1;
 }
 
+// Version holds the version info of components.
+message Version {
+  // Semantic version of the server.
+  string server = 1;
+  // Numeric version of the API.
```

Comment:
Please expand this comment to reflect the comment in `server/api.go`. Maybe even just copy it here.

We expect outside users to read this file, not the Go code.